src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -57,5 +58,7 @@ public class ViewLoadedParams
         //{
             
         //}
+
+        public IEnumerable<WorkspaceViewModel> WorkSpaces;
```

Comment:
Similar to the pattern shown above, let's add a method here `AddMenuItem( MenuBarType, MenuItem )` where `MenuBarType` is an enum that includes `File, View, Edit, Help`.  The method should add the corresponding menu items to the `DynamoView`.